pkg/awsds/sessions_test.go

```diff
@@ -57,7 +58,7 @@ func TestNewSession_AssumeRole(t *testing.T) {
 			AssumeRoleARN: roleARN,
 		}
 
-		cache := NewSessionCache()
+		cache := NewSessionCacheWithSettigns(&AuthSettings{AllowedAuthProviders: []string{"default"}, AssumeRoleEnabled: true})
```

Comment:
Not sure where this function is defined, but it has a typo.
```suggestion
		cache := NewSessionCacheWithSettings(&AuthSettings{AllowedAuthProviders: []string{"default"}, AssumeRoleEnabled: true})
```

Reply:
Oh yes, I'll fix!